component/common.libsonnet

```diff
@@ -19,6 +19,62 @@ local syncOptions = {
   },
 };
 
+local vshnMetaDBaaSExoscale(dbname) = {
+  metadata+: {
+    annotations+: {
+      'metadata.appcat.vshn.io/displayname': 'Exoscale ' + dbname,
+      'metadata.appcat.vshn.io/description': dbname + ' instances hosted by Exoscale',
+      'metadata.appcat.vshn.io/end-user-docs-url': 'https://docs.appuio.cloud/appcat/exoscale-dbaas/' + std.asciiLower(dbname) + '/create.html',
+      'metadata.appcat.vshn.io/zone': 'Exoscale zones: de-fra-1, de-muc-1, at-vie-1, ch-gva-2, ch-dk-2, bg-sof-1',
+      'metadata.appcat.vshn.io/product-description': 'https://products.docs.vshn.ch/products/appcat/exoscale_dbaas.html',
+    },
+    labels+: {
+      'metadata.appcat.vshn.io/offered': 'true',
+      'metadata.appcat.vshn.io/serviceID': 'exoscale-' + std.asciiLower(dbname),
+    },
+  },
+};
+
+local vshnMetaVshn(dbname) = {
+  metadata+: {
+    annotations+: {
+      'metadata.appcat.vshn.io/displayname': 'VSHN Managed ' + dbname,
+      'metadata.appcat.vshn.io/description': dbname + ' instances hosted by VSHN',
+      'metadata.appcat.vshn.io/end-user-docs-url': 'https://docs.appuio.cloud/appcat/vshn-dbaas/' + std.asciiLower(dbname) + '/create.html',
+      'metadata.appcat.vshn.io/zone': 'local cluster',
+      'metadata.appcat.vshn.io/flavor': 'standalone, clustered',
```

Comment:
Shouldn't we make that configurable?

Reply:
This assumes that all services are available in all flavors right away and gives no way to have services, e.g. only in standalone flavor.

Reply:
I updated it to have only standalone version for that. Also added a second argument to the function.